tardis/interfaces/siteadapter.py

```diff
@@ -1,13 +1,37 @@
+from ..configuration.configuration import Configuration
 from ..utilities.attributedict import AttributeDict
 
 from abc import ABCMeta, abstractmethod
+from cobald.utility.primitives import infinity as inf
 from enum import Enum
+from functools import lru_cache
+from pydantic import BaseModel, conint, validator
+from typing import Optional
 
 import logging
 
 logger = logging.getLogger("cobald.runtime.tardis.interfaces.site")
 
 
+class SiteConfigurationModel(BaseModel):
+    """
+    pydantic BaseModel for the input validation of the generic site configuration
+    """
+
+    name: str
+    adapter: str
+    quota: Optional[int] = inf
```

Comment:
Is this actually an optional integer? ``inf`` suggests it is just ``float``.

If it is indeed optional int, a unittest for the optional part would be good.

Reply:
It is an optional integer and the unittest is already in the code. ;-)

https://github.com/giffels/tardis/blob/de103d9b80172ab5740a3e9e14d8302282d83b4d/tests/interfaces_t/test_siteadapter.py#L203-L213

Reply:
Thanks a lot, had not find that. I think it is fine as-is then.